glide.lock

```diff
@@ -82,8 +82,6 @@ imports:
   version: 9208b142303c12d8899bae836fd524ac9338b4fd
 - name: github.com/codegangsta/cli
   version: bf4a526f48af7badd25d2cb02d587e1b01be3b50
-- name: github.com/codegangsta/negroni
-  version: c0db5feaa33826cd5117930c8f4ee5c0f565eec6
```

Comment:
Hum, interesting. I was also expecting to find negroni in the old `glide.yaml`...